HeterogeneousCore/CUDAUtilities/interface/launch.h

```diff
@@ -94,10 +94,14 @@ namespace cudautils {
   }  // namespace detail
 
   // wrappers for cudaLaunchKernel
-
+  inline
   void launch(void (*kernel)(), LaunchParameters config) {
+#ifdef CUDA_KERNELS_ON_CPU
```

Comment:
but I really, really do not want to add a dependency on #ifdefs etc. here.

Reply:
will find a less intrusive solution